lms/static/coffee/src/instructor_dashboard/data_download.coffee

```diff
@@ -140,6 +148,8 @@ class DataDownload
         dataType: 'json'
         url: url
         error: (std_ajax_err) =>
+          if std_ajax_err.status == 401
+            window.location.reload()
```

Comment:
Additionally - what is the reason for special-casing the JSON version of sudo-required with a `401` instead of the standard redirect?